src/test/resources/reference.conf

```diff
@@ -79,7 +79,7 @@ schemaLock {
 
 termsOfService {
   enabled = false
-  version = 0
+  version = 1
```

Comment:
Tests should still test the typical behavior of ToS, which begins at v1. There are a couple of cases I added for v0 that test the specific behavior.